### ISSUE TITLE
First release blockers

### ISSUE THREAD

**Issue by MrTimscampi** [jellyfin/jellyfin-vue#557]
This issue details the needed points to be addressed before we can proceed with the first release.

## Major (Mandatory for release)

* [x] Music player responsiveness (#554, still need addressing for the full screen version)
* [x] Video player overlay when maximized should timeout when the mouse isn't moved (#583)
* [x] The artist page needs a responsive pass (#616)
* [x] Store mutations for position change during playback should be limited to 1 per second (The constant spam of mutations makes performance worse) (#607)
* [x] Add "Resume" and "Play from the beginning" support (#605)
* [ ] Pass the selected source and video/audio/subtitle tracks to the playback manager when playing an item
* [x] Use the ItemMenu component for all "three dots" menus in item pages (#565)
* [ ] Search needs to be finished (#345)
* [x] Onboarding wizard needs to be finished
* [x] Playing single episodes fails
* [ ] Server is sending a WebM stream where Web sends HLS
* [x] Episodes are in the wrong season (#480)
* [ ] Missing images (#711, #769) 
## Intermediate (Nice to have)

* [ ] The light theme needs a color pass (Avoid bright whites, use lighter variants of the dark theme's colors, etc)
* [x] `<nuxt-link>` used for navigation should be able to be copied and the address visible on hover the same way as `<a>` (already fixed here https://github.com/jellyfin/jellyfin-vue/pull/616/commits/44450c2d5ac4b24acd96acadd2f71fa49501fcc0#diff-9ab75b264fc615af11fca1ecc4e5167ea34a4c00d1580c3dfa880ff045276186L22)

## Behind the scenes

* [x] Clean the localization files for real duplicates and those having the same intents

**Comment by ThibaultNocchi** [comment]
I think `Episodes are in the wrong season` can be checked now?

**Comment by ferferga** [comment]
@ThibaultNocchi Yes, done

**Comment by Essasetic** [comment]
Shouldn't ```missing images``` be checked considering #711 has been merged?

**Comment by Camc314** [comment]
Unfortunatly, if you add a single episode in `latest media`, it will show a thumb of the episode instead of the primary image for the series or season. So it's not quite fixed yet
